test/Serilog.Tests/Serilog.Tests.csproj

```diff
@@ -20,6 +20,10 @@
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework.TrimEnd(`0123456789`))' == 'net' ">
```

Comment:
This is from the official documentation: https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2022

But could also be written as `$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net5.0'))` for our use-case.